components/expandable-section/src/expandable-section.ts

```diff
@@ -55,38 +53,31 @@ export class DtExpandableSectionHeader {}
   preserveWhitespaces: false,
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class DtExpandableSection implements CanDisable {
+export class DtExpandableSection implements CanDisable, OnChanges {
   /** Whether the expandable section is expanded. */
   @Input()
   get expanded(): boolean {
     return this._panel.expanded;
   }
   set expanded(value: boolean) {
-    this._panel.expanded = value;
+    this._panel.expanded = coerceBooleanProperty(value);
     this._changeDetectorRef.markForCheck();
   }
 
-  /** Whether the expandable section is disabled. */
+  /** Whether the expandable section is expanded. */
   @Input()
   get disabled(): boolean {
-    return this._panel && this._panel.disabled;
+    return this._panel?.disabled;
```

Comment:
What does optional chaining return if _panel is undefined? Does it return `undefined` or `false`? I'm just curious.

Reply:
undefined but I can cast it to a boolean if you want